pom.xml

```diff
@@ -266,6 +274,7 @@
         <module>integrationtests</module>
         <module>kroxylicious-additional-filters</module>
         <module>kroxylicious-sample</module>
+        <module>systemtests</module>
```

Comment:
I think we have said elsewhere that we want modules to be prefixed.

Reply:
done